lib/Xmds/Soap.php

```diff
@@ -1487,6 +1487,24 @@ protected function doSubmitStats($serverKey, $hardwareKey, $statXml)
             $type = $node->getAttribute('type');
             $duration = $node->getAttribute('duration');
             $count = $node->getAttribute('count');
+            $engagements = [];
+
+            foreach ($node->childNodes as $nodeElements) {
+                /* @var \DOMElement $nodeElements */
+                if ($nodeElements->nodeName == "engagements") {
```

Comment:
`""`'s :cry:

Reply:
I know its pedantic, but we use `'` quotes not double quotes `"` (there are even some above and below these new lines) :smile: